### PR TITLE
Edit PR template and conributing guidelines translation

### PR DESCRIPTION
**Pull request by sarmong**
## CHANGELOG

- Edited PR template translation
- Edited CONTRIBUTING.md translation

See #416 

## Checklist

- [x] I'm familiarized with **[CONTIBUTING guidelines](https://github.com/feature-sliced/documentation/blob/master/CONTRIBUTING.md)**
- [x] I conducted a **[self-review of your changes](https://blog.beanbaginc.com/2014/12/01/practicing-effective-self-review/)**
- [x] I gave a **brief description of the changes**
- [x] I awaited for **CI checks**
- [x] I specified **related materials** if needed (discussion/issue/pull-request...)
- [ ] **All requirements for the related issue are resolved** if issue exists

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
+<!-- 2. Before submitting changes for review, go over the checklist and mark the necessary items at the end -->
 
 <!--
 Our common goal is to reduce review costs and achieve consistency in the code base. 🤙 
 Let's appreciate each other's time =)
 -->
 
-- [ ] I'm familiarized with **[CONTIBUTING guidelines](https://github.com/feature-sliced/documentation/blob/master/CONTRIBUTING.md)**
-- [ ] I conducted a **[self-review of your changes](https://blog.beanbaginc.com/2014/12/01/practicing-effective-self-review/)**
+- [ ] I reviewed **[CONTIBUTING guidelines](https://github.com/feature-sliced/documentation/blob/master/CONTRIBUTING.md)**
```
Comment (azinit) [inline .github/pull_request_template.md:19]: [`nitpick:`](https://conventionalcomments.org/) Да, стало лучше - только тут еще от меня опечатка осталась)
```suggestion
- [ ] I reviewed **[CONTRIBUTING guidelines](https://github.com/feature-sliced/documentation/blob/master/CONTRIBUTING.md)**
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -3,21 +3,21 @@
...
 - Added ...smth...
-- Fix ...smth...
+- Fixed ...smth...
 
 <!-- Screenshots/Screencasts are welcome -->
 
 ## Checklist
 
-<!-- 2. Before submitting changes for review, check the checklist and mark the necessary items at the end -->
+<!-- 2. Before submitting changes for review, go over the checklist and mark the necessary items at the end -->
```
Comment (azinit) [inline .github/pull_request_template.md:12]: [`praise:`](https://conventionalcomments.org/) 🔥 🔥 🔥

**Review by AlexandrHoroshih — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,21 +20,21 @@ First of all, thank you for taking the time to contribute to the project! 👍
...
    > Share your opinion, evaluate the problem identified by the author
-- 🔩 Repeat complex [**issues**][issues]
-   > Some issues are difficult to repeat
-- 🛡️ Conduct a review for [**pull requests**][pr]
+- 🔩 Reproduce complex [**issues**][issues]
+   > Some issues are difficult to reproduce
+- 🛡️ Provide a review for [**pull requests**][pr]
    > Share your opinion and help us with the processing of other people's proposals
-- ⚒️ Suggest [own **pull-requests**!][pr-new]
+- ⚒️ Suggest  our [own **pull-requests**!][pr-new]
```
Comment (azinit) [inline CONTRIBUTING.md:27]: [`question:`](https://conventionalcomments.org/) Typo?
```suggestion
- ⚒️ Suggest your [own **pull-requests**!][pr-new]
```

**Review by azinit — APPROVED** [review]
Стало реально лучше, спасибо! 🚀 🚀

File: CONTRIBUTING.md (hunk 3 above)
Comment (sarmong) [reply CONTRIBUTING.md]: Пофиксил оба, могу мержать?

**Review by RinAkaia — APPROVED** [review]

**Pull request merged**